sk0-back/src/app.controller.ts

```diff
@@ -1,11 +1,19 @@
-import { Controller, Get } from "@nestjs/common";
+import { Body, Controller, Get, Post } from "@nestjs/common";
+import { MailerService } from "./utils/mailer.service";
 
 @Controller()
 export class AppController {
-  constructor() {}
-
+  constructor(private readonly mailer: MailerService) {}
   @Get("/health")
   getHealth() {
     return { health: "ok" };
   }
+
+  @Post("/test-mail")
+  async testMail(@Body("to") to: string) {
+    const subject = "Test mail";
+    const html = "<h1>Test mail</h1><p>This is a test mail</p>";
+    await this.mailer.sendMail(to, subject, html);
+    return { message: "Mail sent" };
+  }
```

Comment:
I am not sure if we would like to expose an unauthenticated endpoint to send any emails, we can hit quotas pretty fast if someone wanted to hit these endpoints.

Reply:
It is only for demonstrational purpose to see how does it work, later on we can totally purge this

Reply:
Coolcool. reolving convo